syndesis/src/modules/data/pages/VirtualizationViewsPage.tsx

```diff
@@ -81,9 +82,9 @@ const sortByName = {
 
 const sortTypes: ISortType[] = [sortByName];
 
-export default class VirtualizationViewsPage extends React.Component<
-  IVirtualizationDetailRouteParams,
-  IVirtualizationDetailRouteState
+export class VirtualizationViewsPage extends React.Component<
+  IVirtualizationViewsPageRouteParams,
+  IVirtualizationViewsPageRouteState
```

Comment:
No need to pass these 2 interfaces to the component.

Reply:
Removed all unneeded interfaces passed to components.